VideoLocker/res/layout/drawer_navigation.xml

```diff
@@ -1,125 +1,129 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/grey_nav_background">
+    xmlns:tools="http://schemas.android.com/tools">
 
-    <FrameLayout
-        android:id="@+id/name_layout"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentTop="true"
-        android:foreground="@drawable/light_selectable_box_overlay">
+        android:layout_height="match_parent"
+        android:background="@color/grey_nav_background">
 
-        <LinearLayout
+        <FrameLayout
+            android:id="@+id/profile_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="@color/grey_name_background"
-            android:gravity="center"
-            android:orientation="horizontal"
-            android:padding="@dimen/edx_margin">
-
-            <de.hdodenhof.circleimageview.CircleImageView
-                android:id="@+id/profile_image"
-                style="@style/profile_image"
-                android:layout_width="@dimen/drawer_profile_image_diameter"
-                android:layout_height="@dimen/drawer_profile_image_diameter"
-                android:layout_marginEnd="@dimen/edx_margin"
-                android:layout_marginRight="@dimen/edx_margin"
-                app:civ_border_width="@dimen/drawer_profile_image_border_width"
-                tools:src="@drawable/profile_photo_placeholder" />
+            android:layout_alignParentTop="true"
+            android:foreground="@drawable/light_selectable_box_overlay">
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="vertical">
-
-                <TextView
-                    android:id="@+id/name_tv"
-                    style="@style/regular_white_text"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:textSize="@dimen/drawer_username_text"
-                    tools:targetApi="17"
-                    tools:text="username" />
-
-                <TextView
-                    android:id="@+id/email_tv"
-                    style="@style/regular_white_text"
-                    android:layout_width="wrap_content"
+                android:background="@color/grey_name_background"
+                android:gravity="center"
+                android:orientation="horizontal"
+                android:padding="@dimen/edx_margin">
+
+                <de.hdodenhof.circleimageview.CircleImageView
+                    android:id="@+id/profile_image"
+                    android:contentDescription="@string/label_my_profile"
```

Comment:
It does indeed behave badly. It doesn't read back the user and email. 

"Added contentdescription to the profile image itself (instead of the whole container) which is visible/invisible based on the config settings."